specifyweb/frontend/js_src/lib/components/WbPlanView/mappingReducer.ts

```diff
@@ -388,9 +388,10 @@ export const reducer = generateReducer<MappingState, MappingActions>({
     }),
     changesMade: true,
   }),
-  UpdateLinesAction: ({ state, action: { lines } }) => ({
+  UpdateLinesAction: ({ state, action: { lines, changesMade } }) => ({
     ...state,
     lines,
+    changesMade,
```

Comment:
like in other reducers in this file, changesMade is always set to true when reducer is called, rather than provided as an argument (i.e see line 389)
thus remove the changesMade argument